internal/nodes/avi_model_evh_nodes.go

```diff
@@ -807,6 +807,8 @@ func (o *AviObjectGraph) ConstructAviL7SharedVsNodeForEvh(vsName, key string, ro
 
 	if infraSetting := routeIgrObj.GetAviInfraSetting(); infraSetting != nil {
 		buildWithInfraSettingForEvh(key, avi_vs_meta, vsVipNode, infraSetting)
+	} else if dedicated {
+		avi_vs_meta.ServiceEngineGroup = lib.GetSEGName()
```

Comment:
isn't the `ServiceEngineGroup` being set in line L760?

Reply:
Nice catch. Fixed at both places.